pypechain/hardhat/utilities.py

```diff
@@ -0,0 +1,20 @@
+"""Utilities for working with foundry-rs."""
```

Comment:
should this be "hardhat" or is "foundry-rs" correct? `hardhat/types.py` says "hardhat"

Reply:
yup yup